### PR TITLE
Removing http-equiv meta tags

### PR DESCRIPTION
**Pull request by batasrki**
Local tests are passing, need to 🎩

### CODE REVIEW

**Comment by kevinhughes27** [comment]
LGTM 🚢  after 🎩

**Comment by batasrki** [comment]
@kevinhughes27 @nwtn This is ready for a last review. I added the description to the CHANGELOG, as well. Let me know if it's sufficient.

**Comment by batasrki** [comment]
References #243

**Comment by kevinhughes27** [comment]
🚀

**Comment by nwtn** [comment]
🚢

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,8 @@
+7.0.6
+-----
```
Comment (Hammadk) [inline CHANGELOG]: @batasrki Isn't Master already on 7.0.8?

Comment (batasrki) [inline CHANGELOG]: Hahahaha. OK, gotta bump everything up.

**Pull request merged**